src/hints/unimplemented.rs

```diff
@@ -84,10 +84,8 @@ pub const ASSERT_END_OF_BYTECODE_SEGMENTS: &str = indoc! {r#"
 };
 
 #[allow(unused)]
-pub const WRITE_ADDRESS: &str = indoc! {r#"
-    from starkware.starknet.core.os.data_availability.bls_utils import split
-
-    segments.write_arg(ids.res.address_, split(ids.value))"#
+pub const ADDITIONAL_DATA_NEW_SEGMENT: &str = indoc! {r#"
+    ids.additional_data = segments.add()"#
```

Comment:
This was removed in a previous PR. Rebase issue?

Reply:
Should have been removed in my merge, and/or I screwed up the merge. Good catch.

f60f9b5